orchestration/templates/create-json-object.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: create-json-object
+spec:
+  templates:
+    - name: main
+      inputs:
+        parameters:
+          - name: release-date
+          - name: archive-path
+          - name: upload-path
+      {{- include "argo.retry" . | indent 6 }}
+      script:
+        image: google/cloud-sdk:slim
+        command: [bash]
+        source: |
+        json='{"release_date":"'{{ "{{inputs.parameters.release-date}}" }}'","archive_path":"'{{ "{{inputs.parameters.archive-path}}" }}'"}'
+        echo "$json" > release_history.json
+        gsutil -m rsync -r /state/release_history.json {{ "{{inputs.parameters.upload-path}}" }}
+        volumeMounts:
```

Comment:
This `volumeMounts` isn't needed any more, and will cause an error now that the `volumes` block has been removed.